build.sbt

```diff
@@ -31,6 +31,7 @@ libraryDependencies  ++=  Seq(
   "com.muquit.libsodiumjna" % "libsodium-jna" % "1.0.4" exclude("org.slf4j", "slf4j-log4j12"),
   "com.github.alanverbner" %% "bip39" % "0.1",
   "ch.megard" %% "akka-http-cors" % "0.3.0",
+  "com.github.scopt" %% "scopt" % "3.7.1",
```

Comment:
Very happy we are finally using scopt. :smile:

Reply:
indeed, yet we might want to use the latest fresh release: http://eed3si9n.com/scopt4 ?

Reply:
Release candidate? I'm excited :)